### ISSUE TITLE
"Applied Accessibility - Wrap Radio Buttons in a fieldset Element for Better Accessibility" cannot be completed in Firefox due to issue with <div> tests

### ISSUE THREAD

**Issue by SuperGeniusZeb** [freeCodeCamp/learn#136]
https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/wrap-radio-buttons-in-a-fieldset-element-for-better-accessibility

This lesson cannot be completed in Firefox because one of the tests ("Your code should not have any `div` tags.") always fails, no matter what the content of the editor is. This issue does not occur in Chromium.

Related:
- #127
- #128
- #129
- #130
- #134

**Comment by Bouncey** [comment]
I cannot reproduce in firefox

**Comment by SuperGeniusZeb** [comment]
I should have clarified, but I was using Firefox Nightly. I recently switched to Firefox Beta, and the issue is still occurring there. I am using Antergos Linux.

I tried disabling various Firefox add-ons, and it turns out that Plasma Integration is the add-on causing the issue:

https://addons.mozilla.org/en-US/firefox/addon/plasma-integration/

Disabling the add-on makes the `<div>` tests immediately start working without even refreshing the lesson page. Likewise, enabling the add-on immediately causes the `<div>` tests to stop working.

I have no idea why this add-on causes an issue. This issue only seems to happen with tests for `<div>`s, not any other HTML element.

The corresponding Chromium extension does not cause any issues with the freeCodeCamp tests. I have no idea if it is because of a difference between Firefox and Chromium, a difference between the Firefox add-on and its matching Chromium extension, both, or neither.

**Comment by raisedadead** [comment]
Hi @SuperGeniusZeb 

Unfortunately we can't support any browser which are not in the standard set. That is if they are not a latest Stable variant, we would not be able to help much. 

Same goes for supporting add-ons, plugins or extensions that are beyond the scope and control of this project.

You are more than welcome to make a pull-request though, if you find a fix for the said browsers.
